crates/ruma-events/src/room/message/sanitize.rs

```diff
@@ -4,7 +4,7 @@
 ///
 /// [rich reply fallback]: https://spec.matrix.org/latest/client-server-api/#fallbacks-for-rich-replies
 pub fn remove_plain_reply_fallback(mut s: &str) -> &str {
-    if !s.starts_with("> ") {
+    if !s.starts_with("> <@") {
```

Comment:
The `@` will fail for some bridges which use the displayname instead of user ID to be user-friendly. Also, `m.emote` fallbacks start with `> * <` rather than `> <`

https://github.com/mautrix/go/blob/main/event/reply.go#L23-L25 has the rules I use

Reply:
Just a precaution, all messages with reply fallbacks are guaranteed to have newlines